pkg/config/config.go

```diff
@@ -150,20 +146,18 @@ type redis struct {
 }
 
 type Authentication struct {
-	Keys                                    keys
 	RefreshTokenSecretKey                   string
 	AccessTokenExpirationSeconds            int
 	RefreshTokenExpirationSeconds           int
 	RefreshTokenRememberMeExpirationSeconds int
 }
 
-type keys struct {
-	PrivateKey string
-	PublicKey  string
-}
-
-func (k keys) GetPrivateKey(logger *slog.Logger) (*rsa.PrivateKey, error) {
-	decode, _ := pem.Decode([]byte(k.PrivateKey))
+func GetPrivateKey(logger *slog.Logger) (*rsa.PrivateKey, error) {
```

Comment:
Currently we're doing this at run time. Could we perhaps do this during "boot time"?